Какие из этих вызовов синтаксически верно сгенерируют исключение?

1. throw "Ошибка"
2. throw new Error("Ошибка")
3. throw { message: "Ошибка" }
4. throw Error("Ошибка")
5. Ни один.

Ответ: 1,2,3,4